jobs/test-override.Jenkinsfile

```diff
@@ -101,8 +113,18 @@ try {
     if (descPrefix == "") {
         if (!bodhi_update_ids.isEmpty()) {
             descPrefix = shwrapCapture("curl -sSL https://bodhi.fedoraproject.org/updates/${bodhi_update_ids[0]} | jq -r .update.title")
-        } else {
+        } else if (!koji_build_ids.isEmpty()) {
             descPrefix = koji_build_ids[0]
+        } else if (!koji_scratch_build_ids.isEmpty()) {
+            descPrefix = shwrapCapture("""
+                koji taskinfo -v -r ${koji_scratch_build_ids[0]} | \
+                grep SRPM: | head -1 | awk -F '/' '{print \$NF}' | sed 's#.src.rpm\$##'
+            """)
+        } else {
+            descPrefix = shwrapCapture("""
+                curl -sSL 'https://copr.fedorainfracloud.org/api_3/build/${copr_build_ids[0]}' | \
+                jq -r '.source_package | \"\\(.name)-\\(.version)\"'
+            """)
```

Comment:
For copr and koji scratch builds let's prefix the description with `scratch:` or `copr:` so it's very clear to people looking at the history for this job that those are not official builds that were tested.